extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/user/ModeledUserContext.java

```diff
@@ -27,8 +27,10 @@
 import com.google.inject.Provider;
 import java.util.Collection;
 import java.util.Date;
+
```

Comment:
Are we intentionally separating imports, now?

Reply:
No, this wasn't an intentional change. I'll remove it.